modules/heat_conduction/src/materials/GapConductance.C

```diff
@@ -214,8 +215,27 @@ GapConductance::GapConductance(const InputParameters & parameters)
 void
 GapConductance::initialSetup()
 {
+  /// set generated from the passed in vector of subdomain names
+  std::set<SubdomainID> subdomain_ids;
+
+  for (auto & name : _subdomain_names)
+    subdomain_ids.insert(_fe_problem.mesh().getSubdomainID(name));
```

Comment:
You can get those with less code by using `BlockRestrictable::blockIDs()`

Reply:
👍